utils/tryte_msg.c

```diff
@@ -0,0 +1,26 @@
+#include "tryte_msg.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define REQ_BODY                                                           \
+  "{\"value\": 0, \"tag\": \"POWEREDBYTANGLEACCELERATOR9\", \"message\": " \
+  "\"%s\", \"address\":\"%s\"}\r\n\r\n"
+
+void gen_rand_trytes(uint16_t len, uint8_t *out) {
+  const char tryte_alphabet[] = "9ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+  uint8_t rand_index;
+  for (int i = 0; i < len; i++) {
+    rand_index = rand() % 27;
+    out[i] = tryte_alphabet[rand_index];
+  }
+}
+
+void gen_tryte_msg(const char *tryte_msg, const uint8_t *addr, char req_body[1024]) {
```

Comment:
Why do we need this function? This function is really a general function, and can be expanded in the place it is called.

Reply:
`get_tryte_msg` is create for two reason:
1. Hidden request body from main function. Putting request body inside main function will cause difficulties for port to new device.
2. Making unified trytes message will let logger and `send_http_msg()` to log and send message more easily.